conftest.py

```diff
@@ -0,0 +1,97 @@
+# This is in the root POCS directory so that pytest will recognize the
+# options added below without having to also specify pocs/test, or a
+# one of the tests in that directory, on the command line; i.e. pytest
+# doesn't load pocs/tests/conftest.py until after it has searched for
```

Comment:
I don't understand the need for this as the `pytest_collection_modifyitems` is altering the tests after collection so it seems like this wouldn't apply anyway. How are you invoking pytest such that this is needed? And is this change somehow related to the `without-hardware` option (it doesn't seem to be) or just bundled in this PR at the same time?

Reply:
I discovered the invoke issue while trying to deal with the problem of running pytest on a unit with real hardware.

How to invoke:
```bash
$ cd $POCS
$ pytest --with-hardware=all
```

Yes, this means I bundled the move of conftest.py with the addition of the new option. Let me know if you want me to split.

Reply:
Let me clarify: the `pytest_addoption` function is the part needed in $POCS/conftest.py, not `pytest_collection_modifyitems`. But, since the latter makes use of the former, it makes sense to keep them together.

Reply:
I invoke `pytest` that way already so not sure why the conftest.py change would make things different. Just trying to understand what it does.